packages/@okta/vuepress-site/docs/concepts/okta-organizations/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Okta Organizations
+---
+
+# Okta Organizations
+An Okta organization (org) is a root object and a container for all other Okta objects. It contains resources such as users, groups, and applications, as well as policy and configurations for your Okta environment.
+
+Within every org, there are users and applications. These are the only mandatory items that must be configured for your org to use Okta. Users can be created in Okta, imported through directory integrations, or imported through application integrations. Applications are connections to public apps (such as Office 365) or proprietary applications (such as your own apps).
+
+## Org URLs
+Okta orgs host pages on subdomains and each org is assigned a URL. The typical org URL is the tenant name (the subdomain), and then the domain name.
+
+**Example**
+companyname.okta.com
+
+**Example EMEA domain URL**
+companyname.okta-emea.com
+
+**Example preview/sandbox domain URL**
+companyname.oktapreview.com
+
+Each organization also has an administrator URL to sign in to the administrator console. The admin URL is the subdomain plus `-admin` (for example, `companyname-admin.okta.com`). If you have [customized your org URL](/docs/guides/custom-url-domain/overview/), access the admin console using your un-customized URL.
+
+You can customize your Okta org URL by replacing the Okta domain name with your own domain name. Using this feature aliases your Okta organization's domain name to another subdomain that you own, like `login.companyname.com`.
```

Comment:
I think we should flip the order of these two paras, since the one above will lead directly into `Administrator Console` below.